src/appleseed.studio/utility/miscellaneous.cpp

```diff
@@ -66,6 +66,7 @@
 #include <QStringList>
 #include <Qt>
 #include <QWidget>
+#include <QResource>
```

Comment:
Order alphabetically.

Actually, I don't know if you need this one, but you do need <QFile> and <QByteArray> (at least).